cloudinary-core/src/main/java/com/cloudinary/api/signing/SigningUtils.java

```diff
@@ -0,0 +1,9 @@
+package com.cloudinary.api.signing;
+
+import com.cloudinary.utils.StringUtils;
+
+class SigningUtils {
+    static String safeString(String str) {
```

Comment:
This can be a method in `StringUtils` instead of a new `SigningUtils` file, as it's not really a signing issue (And may be used by other utils).